frontend/pages/projects/[projectId].js

```diff
@@ -17,46 +17,46 @@ import ConfirmDialog from "../../src/components/dialogs/ConfirmDialog";
 import UserContext from "../../src/components/context/UserContext";
 import PageNotFound from "../../src/components/general/PageNotFound";
 
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles((theme) => ({
```

Comment:
Just lint and Prettier fixes in this file (like adding parentheses around params). Happy to revert if undesired.